repository.yaml

```diff
@@ -0,0 +1 @@
+repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
```

Comment:
What is this for? Is it required?

Reply:
Sorry that's an HMRC construct that we use in all our public repos. Not required here, will remove.

Reply:
Removed.